src/v/security/audit/schemas/application_activity.h

```diff
@@ -48,12 +48,14 @@ class api_activity final : public ocsf_base_event<api_activity> {
       : ocsf_base_event(
         category_uid::application_activity,
         class_uid::api_activity,
+        ocsf_redpanda_metadata_cloud_profile(),
```

Comment:
nit: Could use useful to link the github issue to the bug as a comment or in the commit message